.travis.yml

```diff
@@ -1,23 +0,0 @@
-services:
```

Comment:
Why?

Reply:
When I checked out to my UI feedback branch from the develop branch which has all those files from my visual studio code, I noticed those API files where still present but I ignored them since they were marked as Untracked, but I realized later after pushing the code to the feedback branch that they where pushed as well, so I had to remove them to keep the UI code clean

Reply:
gotcha.